docs/BigQueryMultiTable-batchsink.md

```diff
@@ -36,8 +36,22 @@ It will be automatically created if it does not exist, but will not be automatic
 Temporary data will be deleted after it is loaded into BigQuery. If it is not provided, a unique
 bucket will be created and then deleted after the run finishes.
 
-**Split Field:** The name of the field that will be used to determine which table to write to. 
-Defaults to `tablename`.
+**Split Field:** The name of the field that will be used to determine which table to write to.
+
+**Update Table Schema**: Whether the BigQuery table schema should be modified 
+when it does not match the schema expected by the pipeline. 
+* When this is set to false, any mismatches between the schema expected by the pipeline 
+and the schema in BigQuery will result in pipeline failure. 
+* When this is set to true, the schema in BigQuery will be updated to match the schema 
+expected by the pipeline, assuming the schemas are compatible. 
+
+Compatible changes fall under the following categories:                
+* the pipeline schema contains nullable fields that do not exist in the BigQuery schema. 
+In this case, the new fields will be added to the BigQuery schema.
+* the pipeline schema contains nullable fields that are non-nullable in the BigQuery schema. 
+In this case, the fields will be modified to become nullable in the BigQuery schema.
```

Comment:
add:
the pipeline schema does not contain fields that exist in the BigQuery schema. In this case, those fields in the BigQuery schema will be modified to become nullable.

Reply:
Done.